dht.go

```diff
@@ -216,6 +216,8 @@ func New(ctx context.Context, h host.Host, options ...Option) (*IpfsDHT, error)
 	// handle providers
 	dht.proc.AddChild(dht.ProviderManager.Process())
 
+	dht.fixLowPeers()
```

Comment:
@aschmahmann fixLowPeers entails a blocking operation (Host.Connect). But New() and Bootstrap() (below) are non-blocking by definition. In both cases, we need a shim that runs fixLowPeers in a separate goroutine and when it is ready, it runs the refresh manager.